gateway/webrtc/client.js

```diff
@@ -60,6 +61,37 @@ $(document).ready(function() {
     connectToSignallingJMS();
 });
 
+function showInCall() {
+    callBtn.style.display='none';
+    callToUsernameInput.style.display='none';
+    hangUpBtn.style.display='inline';
+    overlay.style.visibility='visible';
+    overlay.innerText='Connected to '+connectedUser;
+}
+
+function showOffCall() {
+    callBtn.style.display='inline';
+    callToUsernameInput.style.display='inline';
+    hangUpBtn.style.display='none';
+    overlay.style.visibility='hidden';
+}
+
+function reconstructSDP (sdp) {
+    var result = [];
+
+    var lines = sdp.split("\n");
+    for (let x of lines) {
+        if (x.indexOf("candidate") > 0 && x.indexOf("typ host") > 0) {
```

Comment:
This is good work,

> Otherwise we would expose the local IP via signalling

This is slightly different from the requirement of the hiding of the reflexive transport address, and a kaazing user may or may not care about it.

Questions, without this code does the demo still always go through the relay?  If yes, please surround this logic (and related code below) in a constant flag with something like "HideHostAddress" and set Default to true.  (making this a configurable variable on the UI would be ok too).

If not, please surround with the same constant referenced here: https://github.com/kaazing/webrtc/pull/25/files/b25fd85759d54959e17f7d0de7e62d9542edf421#r86050707